### PR TITLE
Fix token specification

### PR DESCRIPTION
**Pull request by awkure**
Several problems have arised with current token specification:
  1. Since CENTRE token specification describes operations
that operate on a single token type and we have decided to stick
to it.
  2. Some types are ambiguous.
  3. Whitelisting and blacklisting is undocumented.
  4. Some typos.

## Related issue(s)

Resolves #

#### Related changes (conditional)

- Tests
  - [X] If I added new functionality, I added tests covering it.
  - [X] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [X] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [X] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [X] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

**Comment by awkure** [comment]
wtf

File: docs/specification.md [hunk 1]
```diff
@@ -89,12 +89,18 @@ its result.
...
 
 transferParam
   ( address :from_
   , address :to_
   , tokenId :tokenId
+=======
+transferParam
+  ( address :from_
+  , address :to_
+>>>>>>> Fix specification
```
Comment (gromakovsky) [inline docs/specification.md:103]: :fffuuu:

Comment (awkure) [reply docs/specification.md]: Thank you github, very cool

Comment (awkure) [reply docs/specification.md]: Ok, I guess it's up to you to complete this MR

File: docs/specification.md [hunk 2]
```diff
@@ -89,15 +89,11 @@ its result.
 
 Types
 ```
-tokenId = nat
-
 transferParam
   ( address :from_
   , address :to_
-  , tokenId :tokenId
```
Comment (gromakovsky) [inline docs/specification.md:97]: The type of `transfer` is specified in FA2, we can't change it.

Comment (gromakovsky) [reply docs/specification.md:97]: The same applies to other methods from FA2.

File: docs/specification.md (hunk 1 above)
Comment (gromakovsky) [reply docs/specification.md]: Ok, let me deal with the specification.

**Pull request closed without merging**